### PR TITLE
Adds sane player limits to all maps

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Adds sane player limits to all maps. Check the file edited for minimum values and maximum values. Values of 0 have no limits.

## Why It's Good For The Game

Some maps needed player limits. It's insane that we can get very large maps with a very low player amount, and we can get very small maps with a very high player amount.

Judgement on whether or not a map is a small (40), medium (60), or large (80) sized map was based on these statistics.
https://ss13stats.skullnet.me/server/59d29347c391ee6067561615d7370e5061417206480b697ce49119c99299a1f7

## Proof Of Testing

If it compiles, it werks.

## Changelog

:cl: BurgerBB
add: Adds sane player limits to all maps.
/:cl:

### CODE REVIEW

**Comment by BurgerLUA** [comment]
I was 10000% sure I put this on the fucking config branch but whatever I don't think the config branch is active

**Comment by Swiftfeather** [comment]
> I was 10000% sure I put this on the fucking config branch but whatever I don't think the config branch is active

I've asked TGS havers and tainers to go ahead and force update the config repo with current server config but it hasn't quite happened yet

File: config/maps.txt [hunk 1]
```diff
@@ -16,33 +16,40 @@ endmap
...
+	maxplayers 0
 endmap
 
 map metastation
-	minplayers 25
-	#voteweight 0.5
+	default
 	votable
+	minplayers 0
+	maxplayers 80
```
Comment (Iamgoofball) [inline config/maps.txt:40]: why does meta have an 80 playercap? it's equal with the other main stations.

Comment (BurgerLUA) [reply config/maps.txt:40]: Meta feels very small. I could be wrong.

**Comment by Swiftfeather** [comment]
Put this on the config branch, it's not as active as it should be but that doesn't mean we can't make PRs and bug TGS havers to use it.

**Pull request closed without merging**